lib/IRGen/IRGen.cpp

```diff
@@ -797,10 +797,9 @@ static void embedBitcode(llvm::Module *M, const IRGenOptions &Opts)
       reinterpret_cast<const uint8_t *>(OS.str().data()), OS.str().size());
   llvm::Constant *ModuleConstant =
     llvm::ConstantDataArray::get(M->getContext(), ModuleData);
-  llvm::GlobalVariable *GV = new llvm::GlobalVariable(*M,
-                                       ModuleConstant->getType(), true,
-                                       llvm::GlobalValue::PrivateLinkage,
-                                       ModuleConstant);
+  llvm::GlobalVariable *GV = new llvm::GlobalVariable(
+      *M, ModuleConstant->getType(), true, llvm::GlobalValue::PrivateLinkage,
+      ModuleConstant, ".swiftmoduleconstant");
```

Comment:
Can we come up with a better name?  Even `.constant` is better IMO.  These do get altered as necessary by `llvm::Module`.